src/dataflow/components/nodes/controls/sensor-select-control.tsx

```diff
@@ -26,45 +27,82 @@ export class SensorSelectControl extends Rete.Control {
                                    value: any;
                                    onTypeChange: any;
                                    onSensorChange: any;
+                                   onSensorClick: any;
+                                   onListClick: any;
+                                   showList: any
```

Comment:
It's a preexisting condition, so arguably not part of this PR, but surely we can do better than `any` for typing the properties of our own components.

Reply:
Added some changes to fix this.